templates/show-task.html

```diff
@@ -36,13 +39,17 @@
 
 {% block content %}
 	<div class="container">
+        <div id="alert-message-holder"></div>
 		<div class="row">
 			<div class="col-md-12 create-worklist-container">
-				<h2 class="top-heading">Worklist Name: {{worklist_name}}</h2>
+                <h2 class="top-heading">Worklist Name: {{worklist_name}}</h2>
+                <a href="#" data-toggle="popover" data-content="Share Worklist using this link: https://tools.wmflabs.org/worklist-tool/show-tasks/{{ worklist_created_by }}/{{ worklist_name }}"><i class="fa fa-share-alt" class="share-icon-size"></i></a>
```

Comment:
Personal preference, but I think I'd make the share button larger (people tend to not find small buttons) and possibly align it to the right, above the Search button.

Reply:
Done!